frontend/internal-packages/db/supabase/migrations/20250616065500_add_validation_tables.sql

```diff
@@ -0,0 +1,281 @@
+BEGIN;
+
+-- Create validation_queries table
+CREATE TABLE IF NOT EXISTS "public"."validation_queries" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "design_session_id" "uuid" NOT NULL,
+    "query_string" "text" NOT NULL,
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    "updated_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+ALTER TABLE "public"."validation_queries" OWNER TO "postgres";
+
+-- Create validation_results table
+CREATE TABLE IF NOT EXISTS "public"."validation_results" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "validation_query_id" "uuid" NOT NULL,
+    "result_set" "text",
+    "executed_at" timestamp with time zone DEFAULT "now"() NOT NULL,
+    "status" "text" NOT NULL,
+    "error_message" "text",
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    "updated_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    CONSTRAINT "validation_results_status_check" CHECK (("status" IN ('success', 'failure')))
```

Comment:
(aside)
(ask)

@NoritakaIkeda 
Why isn't status defined as a boolean (true/false)?
Is it for potential future extensibility?